app/src/main/java/woowacourse/payments/ui/addcard/component/AddCardVendorModal.kt

```diff
@@ -0,0 +1,87 @@
+package woowacourse.payments.ui.addcard.component
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.FlowRow
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.ModalBottomSheet
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import woowacourse.payments.model.CardVendor
+import woowacourse.payments.ui.addcard.model.VendorModalUiState
+import woowacourse.payments.ui.theme.AndroidpaymentsTheme
+import woowacourse.payments.ui.uimodel.VendorUiModel
+import woowacourse.payments.ui.uimodel.toUiModel
+
+@Composable
+@OptIn(ExperimentalMaterial3Api::class)
+fun AddCardVendorModal(
+    vendorModalUiState: VendorModalUiState,
+    onDismissRequest: () -> Unit = {},
+    onVendorItemClick: (VendorUiModel) -> Unit = {},
```

Comment:
```suggestion
    onVendorItemClick: (uiModel: VendorUiModel) -> Unit = {},
```
p2) 람다로 넘기는 변수들도 변수명을 지정해줄 수 있는 것을 아시나요? 저는 안지 얼마 되지 않았어요 👀

Reply:
오 ! 이런 방법이 있었군요 !
다음 미션에 반영하겠습니다 ~!